### ISSUE TITLE
Hide denomination token (USDC) balance when not applicable

### ISSUE THREAD

**Issue by kenkunz** [tradingstrategy-ai/frontend#650]
# Issue

If a user clicks "connect wallet" from a strategy that doesn't have deposits enabled, on the "Wallet balance" wizard step, the USDC balance is displayed as "---" even if the user has USDC in their wallet. In addition, the "Next" button is disabled, preventing the user from completing the wizard.

## Steps to reproduce

1. Navigate to a strategy that has deposits disabled, e.g., [ETH mean reversion bounce](http://localhost:5173/strategies/polygon-eth-spot-short)
2. Click "Connect wallet" in the header (you may need to disconnect first)
3. Connect a wallet that includes a positive USDC balance
4. Advance to the "Wallet balance" step
5. Note the USDC balance
   - expected: user's USDC balance for the connected wallet
   - actual: "---"
 6. Note the "Next" button state
    - expected: enabled
    - actual: disabled
 
<img width="500" alt="image" src="https://github.com/tradingstrategy-ai/frontend/assets/35901/0a8d7ffc-63c5-48f5-94c3-22bd479c3d71">

## Cause

The wallet balances displayed on that screen are based on (a) the native currency for the chain; (b) the "denomination token" returned by the "comptroller" smart contract that's configured for the strategy. Since this strategy is hot-wallet based, there are no smart contracts configured:

```json
"on_chain_data": {
   "chain_id": 137,
   "asset_management_mode": "hot_wallet",
   "smart_contracts": {}
}
```

## Recommended fix

If user initiates "connect wallet" wizard from a strategy that doesn't have deposits enabled, hide the "denomination token" (aka USDC) row in the "Wallet balance" step / table. Also, enable the "Next" button once the native currency balance has loaded.

**Comment by kenkunz** [comment]
fixed by #662

**Comment by kenkunz** [comment]
fixed by #662
